### ISSUE TITLE
Remove table providerpaymentpreference

### ISSUE THREAD

**Issue by situatedbit** [loconomics/loconomics#142]
``providerpaymentpreference`` table has no dependencies in the database (other than the stored procedure ``DeleteUser``). It is referenced only once in the codebase in a read statement. It appears to not actually be used. 

@IagoSRL I recommend removing it. 

Additionally, ``providerpaymentpreferencetype`` is no longer necessary either.

**Comment by IagoSRL** [comment]
You are right. This is part of an obsolete code, before Braintree offered the Marketplace feature. Now is not needed, and we don't want to go back.

Is perfect to remove that two tables:
- providerpaymentpreference
- providerpaymentpreferencetype

And dependencies as you mentioned, I could find:
- Method LcPayment.CreateProviderPaymentAccount(int providerID, BraintreeGateway gateway = null)
- Method LcPayment.CreateProviderPaymentAccount(dynamic user, LcData.Address address, string ABANumber, BraintreeGateway gateway = null)
- **NOTE**: there is a third overload method for LcPayment.CreateProviderPaymentAccount that must be kept, is the only one we use currently.
- File /web/tests/TestBraintreeMarketplace.cshtml
- LcData.UserInfo.GetUserBankInfo

@joshdanielson If you have some statement on this, or disagree, this is the time :-)

**Comment by joshdanielson** [comment]
@iagosrl as long as the user can access their bank info to update it. I'm not sure how updates to Braintree work. @situatedbit and I can review Monday

**Comment by IagoSRL** [comment]
@joshdanielson: all bank info is saved at Braintree now (what user adds at the paymentAccount activity).

Before the 'Braintree Marketplace' we had to save some (sensitive) data in our database, and need methods to update and read that data; we were holding money from clients and then (manually) sending to professionals, in the way they have chosen (the paymentpreferencetype: checking, savings, paper check, bank account). Now Braintree sends money directly to professionals.

Take a look at the tables if helps to refresh memory (this is old, and *so many things*).
